### PR TITLE
Grey box update

### PR DESCRIPTION
**Pull request by sdraper-CS**
Just doc for payload formats to support grey box

### CODE REVIEW

File: docs/instructions.md [hunk 1]
```diff
@@ -61,11 +61,34 @@ curl -X POST \
...
+`theshold` is ignored in this case.  The model may optionally return the ordering of the `scores` array (label dimension)
+overriding anything specified in the Certifai scan definition.  This allows models to assert whatever label
+ordering metadata is appropriate to them in a way that ensures compatability with the particular deployed
+trained instance of the model.
+```
+{
+   "payload": {
+     "predictions": [3],
+     "scores": [[0.2, 0.2, 0.2, 0.4]],
+     "labels": ["one:, "two", "three"]
```
Comment (cdraper-CS) [inline docs/instructions.md]: errant ":"  after one and missing quote!

**Comment by cdraper-CS** [comment]
Assume this replaces the other PR

**Comment by sdraper-CS** [comment]
@akumar-cs @cdraper-CS Rebased on master and typos corrected.  Please let me know if ok to merge

**Comment by sdraper-CS** [comment]
Superceded by another PR

**Pull request merged**